assets/js/index.js

```diff
@@ -154,6 +154,16 @@ $(document).ready(function () {
     }
   });
 
+  $("body").click(function(event) {
+    let sidebar = $(".sidebar");
+    if(event.target.className.split(" ",1)!="sidebar" && event.target.className.split(" ",1)!="fas" && sidebar.hasClass("animate__slideInLeft") && (!sidebar.hasClass("hide-sidebar"))){
```

Comment:
before this if condition, add a new variable `eventTarget = $(event.target)` and then in the if condition use functions like `hasClass` instead of split

Reply:
i found ```hasclass``` not working properly so i used ```split```

Reply:
> before this if condition, add a new variable `eventTarget = $(event.target)` and then in the if condition use functions like `hasClass` instead of split

we can't do ```eventTarget = $(event.target)  ``` because on doing so it is throwing an error ``` split() not define```

Reply:
> > before this if condition, add a new variable `eventTarget = $(event.target)` and then in the if condition use functions like `hasClass` instead of split
> 
> we can't do `eventTarget = $(event.target) ` because on doing so it is throwing an error ` split() not define`

i tried this before but it is throwing an error..!!